.github/workflows/pipeline.yml

```
@@ -14,7 +14,7 @@ on:
 env:
   DUMMY: 3 # For cache busting.
   NODE_VERSION: 18.16.0
-  RUST_VERSION: 1.73
+  RUST_VERSION: 1.74
```

Needs to be at least
```suggestion
  RUST_VERSION: 1.78
```
to be compatible with lock version 4 (from `cargo.lock`). I think we should go with the highest version you can get it to build with though, i.e. if it works with version 1.85 on your machine, let's specify that here as well.